data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -26,40 +27,45 @@
 import urllib
 import pandas as pd
 
+from google.oauth2 import service_account
+
 pd.options.display.max_rows = 120
+
+creds = service_account.Credentials.from_service_account_file(key_file)
+
 # -
 
 # # Table comparison
 # The export should generally contain the same tables from month to month. Tables found only in the old or the new export are listed below.
 
 query = f'''
-SELECT 
+SELECT
   COALESCE(curr.table_id, prev.table_id) AS table_id
- ,curr.row_count AS {new_rdr}
- ,prev.row_count AS {old_rdr}
- ,(curr.row_count - prev.row_count) row_diff
-FROM `{project_id}.{new_rdr}.__TABLES__` curr 
+ ,curr.row_count AS _{new_rdr}
+ ,prev.row_count AS _{old_rdr}
+ ,(curr.row_count - prev.row_count) AS row_diff
+FROM `{project_id}.{new_rdr}.__TABLES__` curr
 FULL OUTER JOIN `{project_id}.{old_rdr}.__TABLES__` prev
   USING (table_id)
 WHERE curr.table_id IS NULL OR prev.table_id IS NULL
 '''
-pd.read_gbq(query, dialect='standard')
+pd.read_gbq(query, dialect='standard', credentials=creds)
```

Comment:
As of `pandas-gbq 0.10.0` the param `dialect` _should_ default to `standard` ([docs](https://pandas-gbq.readthedocs.io/en/stable/changelog.html#changelog-0-10-0)) but for some reason here it defaults to `legacy` even though our [requirements.txt ](https://github.com/all-of-us/curation/blob/develop/data_steward/requirements.txt#L65) says `pandas-gbq==0.11.0`. Any idea what this is about? It would be nice to eliminate duplicate code.

Reply:
@mark-velez According to the [read_gbq docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_gbq.html), the default for this function is legacy and will become standard in a future release.

Reply:
Thanks @lrwb-aou ; sorry, I'm still confused (not your fault), could you help me understand? Is the `read_gbq` being used different from [this one](https://github.com/pydata/pandas-gbq/blob/0.11.0/pandas_gbq/gbq.py#L856) in `pandas-gbq==0.11.0`? :confused:

Reply:
@mark-velez it's a confusing mess.   But, the link I posted is `pandas.read_gbq` which uses `pandas_gbq.read_gbq`.  Your link is the `pandas_gbq` link.  I didn't write the `dialect='standard'` code, so I don't want to change it without an investigation.  But, looking at the notebook, the code is using the `pandas.read_gbq` function which defaults to legacy and passes that to `pandas_gbq.read_gbq`. 

We could just make the whole thing a lambda function that gets defined once and called multiple times.  But that also feels outside the scope of this PR.  I just wanted the thing to run on rdr datasets whose:
1. names start with numbers
2. add backticks to fully qualified table names
3. add descriptive comments
4. add a smoking rule check to make sure we don't lose that data.

Basically, I just want to increase it's usability right now.